---a/net/minecraft/block/BlockCompressed.java
+++b/net/minecraft/block/BlockCompressed.java
@@ -5,21 +5,19 @@
 import net.minecraft.block.state.IBlockState;
 import net.minecraft.creativetab.CreativeTabs;
 
-public class BlockCompressed extends Block
-{
+public class BlockCompressed extends Block {
+
     private final MapColor mapColor;
     private static final String __OBFID = "CL_00000268";
 
-    public BlockCompressed(MapColor mapColorIn)
-    {
+    public BlockCompressed(MapColor mapColorIn) {
         super(Material.iron);
         this.mapColor = mapColorIn;
         this.setCreativeTab(CreativeTabs.tabBlock);
     }
 
-    public MapColor getMapColor(IBlockState state)
-    {
+    public MapColor getMapColor(IBlockState state) {
         return this.mapColor;
     }
-}
+}
